Dockerfile

```diff
@@ -0,0 +1,20 @@
+# This file is part of DataCite.
+#
+# Copyright (C) 2015 CERN.
+#
+# DataCite is free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+# docker build .
+FROM python:3.3
+RUN pip install pep257 "coverage<4.0a1"
+RUN pip install pytest pytest-pep8 pytest-cov pytest-cache
+RUN pip install mock httpretty requests
+RUN pip install sphinx_rtd_theme
```

Comment:
One command would generate less image layers, and we should use CMD, but I can send another PR with some Docker improvements later.